pymc/sampling/jax.py

```diff
@@ -74,6 +74,15 @@ def assert_fn(value, *inps):
     return assert_fn
 
 
+@jax_funcify.register(PosDefMatrix)
+def jax_funcify_PosDefMatrix(op, **kwargs):
+    def posdefmatrix_fn(value, *inps):
+        no_pos_def = jnp.any(jnp.isnan(jnp.linalg.cholesky(value)))
+        return no_pos_def == jnp.array(False, dtype="bool")
```

Comment:
Isn't this enough?

```suggestion
        return not no_pos_def
```

Reply:
I would make it a one liner because the double negation with the name is confusing